### PR TITLE
Support spaces in app path

### PR DESCRIPTION
**Pull request by kukicola**
### Summary

Hello, when upgrading my app from v6 to v7, I noticed some commands are failing because I have a space in the absolute path of my project. 

### Pull Request checklist

- [x] Add/update test to cover these changes
- [x] ~Update documentation~
- [x] Update CHANGELOG file

### Other Information

### CODE REVIEW

**Comment by ahangarha** [comment]
May you create a minimal app, demonstrating the issue?

**Comment by kukicola** [comment]
Easiest way to replicate the issue is to create a new app in the folder with space (in this case `test with space`), add shakapacker, and execute `shakapacker:install` task, which will fail:

```
➜  test with space pwd
/Users/kukicola/Desktop/test with space
➜  test with space rails new . --skip-javascript
       exist  
      create  README.md
      create  Rakefile
      create  .ruby-version
      create  config.ru
      ...

➜  test with space git:(main) ✗ bundle add shakapacker --strict
Fetching gem metadata from https://rubygems.org/..........
Resolving dependencies...
Fetching gem metadata from https://rubygems.org/..........
...

➜  test with space git:(main) ✗ ./bin/rails shakapacker:install
/Users/kukicola/Desktop/test: --> /Users/kukicola/Desktop/test
syntax error, unexpected integer literal, expecting end-of-input
```

**Comment by ahangarha** [comment]
Do you really need to have your app in a directory with such name? This is not considered a good practice to name files or directories with spaces.

**Comment by justin808** [comment]
@ahangarha please test this and I'll ship 7.0.3 with this. Please update the changelog.

File: CHANGELOG.md
Comment (ahangarha) [inline CHANGELOG.md:1]: Please include link to the PR and your profile (similar to other entries)

File: spec/generator_specs/generator_spec.rb
Comment (ahangarha) [inline spec/generator_specs/generator_spec.rb:1]: If we want to support spaces in the path for tests, there are more cases to be updated.

**Review by ahangarha — APPROVED** [review]
Thanks for the PR.

I tested your changes and it works as expected.
I added few comments. Take a look at them please.

File: CHANGELOG.md
Comment (kukicola) [reply CHANGELOG.md:1]: done

**Comment by ahangarha** [comment]
@kukicola 
Please rebase your branch on the latest changes on our master branch.

@justin808 
Except for Changelog, this PR looks good to me.

**Comment by ahangarha** [comment]
@kukicola I don't think you did it in the right way.
If you don't mind, I will bring your commits and create a new PR and fix the changelog.

@justin808 do you have any other consideration?

**Comment by justin808** [comment]
@ahangarha I merged. You can add another commit and I can release a new version.

**Pull request merged**